Styles/riveraStyle.css

```diff
@@ -0,0 +1,282 @@
+      font-weight: 900;
+      /* margin-top: 15px; */
+      width: 240px;
+      /* height: 81px; */
+  }
+
+  /* .navbar {
+      height: 60px;
+  } */
+
+  .no-margin {
+  margin: 0px !important;
+}
+
+.firstDiv {
+    background-color: black;
+    height: 35px;
+}
+
+.navbar {
+    padding-top: 5px;
+    padding-bottom: 5px;
+}
+
+.nav-link {
+    padding-left: 2px;
+}
+
+.navbar-brand {
+    font-size: 30px;
+    font-weight: bolder;
+}
+
+/* this position relative then text is position absolute to pic */
+.carousel-item {
+    position: relative;
+    text-align: center;
+    color: white;   
+}
+
+.carousel-control-prev-icon {
+    background-color: #ED564C;
+}
+
+.carousel-control-next-icon {
+    background-color: #ED564C;
+}
+
+.centered {
+    color: white;  
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+}
+
+.centered > h3 {
+    font-size: 30px;
+    text-transform: uppercase;
+    font-weight: 100;
+    color: #dadcdd;
+    opacity: 0.7;
+}
+
+.centered > p {
+    font-size: 12px;
+    color: #dadcdd;
+    opacity: 0.5;
+}
+
+/* turn imagees black and white */
+.d-block  {
+    -webkit-filter: grayscale(80%); /* Safari 6.0 - 9.0 */
+    filter: grayscale(100%);
+}
+
+/* .doverlay {
+background: url(starter-code/img/overlay.png);
+    background-image: url(starter-code/img/overlay.png);
+    background-position-x: initial;
+    background-position-y: initial;
+    background-size: initial;
+    background-repeat-x: initial;
+    background-repeat-y: initial;
+    background-attachment: initial;
+    background-origin: initial;
+    background-clip: initial;
+    background-color: initial;
+    position: absolute;
+    width: 100%;
+    height: 100%;
+    left: 0px;
+    top: 0px;
+    z-index: 100;
+} */
+
+.red {
+    color:  #ED564C;
+}
+  
+.widgets {
+    color: #f8f9fa;
+    background-color: #ED564C;
+    width: 100%;
+    text-align: center;
+    padding-top: 20px;
+    padding-bottom: 20px;
+}
+
+.homewidget {
+    font-size: 24px;
+    font-weight: bold;
+    vertical-align: baseline;
+    /* box-sizing: border-box;
+    display: block;
+    margin-block-start: 1em;
+    margin-block-end: 1em; */
+    /* margin-inline-start: 0px;
+    margin-inline-end: 0px;  */
+}
+
+.icon > p {
+    font-size: 10px;
+    font-weight: 100;
+}
+
+/* circle around the glyphs */
+.fas {
+    border-radius: 50%;
+    border: 3px solid #f8f9fa;
+    padding: 15px;
+    margin-bottom: 10px;
+}
+
+.bannerLastDiv{
+    padding-right: 0;
+}
+
+.projectsContainer {
+    background-color: #f8f9fa;
+}
+
+.titlePtags {
+    color: #929394;
+}
+
+.section-title {
+    padding-left: 0;
+    padding-right: 0;
+}
+.projectsSection {
+    width: 100%;
+    padding-top: 45px;
+}
+
+.projectsText > h2 {
+    font-size: 20px;
+    font-weight: bolder;
+}
+
+.projectsText > p {
+    font-size: 13px;
+}
+
+.articlesContainer {
+    background-color: #f8f9fa;
+}
+
+.imgContainer {
+    display: flex;
```

Comment:
Bootstap has built in selectors that take care of displaying flex. 
This could be valuable to check out.
https://getbootstrap.com/docs/4.4/utilities/flex/